apps/frontend/core-dapp/src/components/SummaryRecord.tsx

```diff
@@ -36,7 +36,9 @@ const SummaryRecord: React.FC<Props> = ({ children, className, label, loading, t
       <Flex gap={4}>
         <LabelText>{label}</LabelText>
       </Flex>
-      <ValueCol>{loading ? <Skeleton height="22px" width="64px" /> : children}</ValueCol>
+      <ValueCol className="summary-record-value">
```

Comment:
did you mean data-testid or is this intended?

```suggestion
      <ValueCol data-testid="summary-record-value">
```

Reply:
This is intended, because there can be multiple summary records in the same screen, which would result in multiple elements with the same test ID. The intention is to find the parent element by test ID and then find the value by class name.